src/concerns/focus.coffee

```diff
@@ -0,0 +1,28 @@
+###
+Code related to focus and hover state
+###
+export default
+
+	# Some simple data about our component and window its mounted on
+	data: ->
+		hovered: false
+		buttonFocused: false
```

Comment:
I don't think this is used, right?

```suggestion
```

Reply:
Correct, I'll remove for the time being, I was starting to look at tab-focusing for buttons where we also need to pause

Reply:
Oooh fancy! I like where your head is at with that.

Reply:
Yeah, I figure it may be needed for some ADA compliance..